### PR TITLE
Add cron lambda that enforces log retention policy.

### PR DESCRIPTION
**Pull request by mweiden**
This lambda runs every two days and checks the retention policies of all
log groups. If the log group retention policy is not specified in the
lambda's configuration file, it sets it to a default TTL of 90 days. If
the log group retention policy is specified, it sets it to that TTL.

### CODE REVIEW

File: apps/log_retention_policy_enforcer/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Delete Old ES Indices
```
Comment (parthshahva) [inline apps/log_retention_policy_enforcer/README.md]: This readme looks incorrect

Comment (mweiden) [reply apps/log_retention_policy_enforcer/README.md]: Ah yes, missed this in the copy. One sec and I'll fix this.

**Comment by mweiden** [comment]
@parthshahva this is finally 🍏 Approve?

**Comment by mweiden** [comment]
@parthshahva it should!

**Pull request merged**